src/Hecke.jl

```diff
@@ -111,6 +111,18 @@ import Nemo: acb_struct, Ring, Group, Field, zzModRing, zzModRingElem, arf_struc
 
 const RationalUnion = Union{IntegerUnion, Rational{<: Integer}, QQFieldElem}
 
+const solve = AbstractAlgebra.Solve.solve
+const can_solve = AbstractAlgebra.Solve.can_solve
+const can_solve_with_solution = AbstractAlgebra.Solve.can_solve_with_solution
+
+function kernel(M::MatElem; side::Symbol = :right)
+  return AbstractAlgebra.Solve.kernel(M, side = side)
+end
+
+function kernel(R::Ring, M::MatElem; side::Symbol = :right)
+  return AbstractAlgebra.Solve.kernel(R, M, side = side)
+end
+
```

Comment:
This is the "non-trivial" part of this pull request. Is this fine or should I also make dummy functions for `solve` etc.? (It's only temporarily anyways.)

Reply:
Yes, this is fine